circuit/environment/Cargo.toml

```diff
@@ -69,6 +63,3 @@ path = "../../console/types"
 
 [dev-dependencies.criterion]
 version = "0.4"
-
-[features]
-default = [ "snarkvm-curves/default", "snarkvm-r1cs/default" ]
```

Comment:
I'd suggest to leave the `snarkvm-curves/default` and remove the `snarkvm-r1cs/default` perhaps